packages/@react-spectrum/tooltip/test/TooltipTrigger.test.js

```diff
@@ -34,81 +33,6 @@ describe('TooltipTrigger', function () {
     cleanup();
   });
 
-  describe('click related tests', function () {
```

Comment:
Why did you remove these tests?

Reply:
Adding some tests for focus behavior is likely the only change that needs to be made to the tests.

Reply:
We are not supporting click functionality for Tooltips. They should only open on hover and focus interactions so we won't need these tests.